scripts/quota_log.py

```diff
@@ -35,12 +47,19 @@ def disk_quota():
 
 def cpu_hours():
     current_time = datetime.datetime.now()
-
-    # script that runs on uppmax
-    uq = subprocess.Popen(["/sw/uppmax/bin/projinfo", '-q'], stdout=subprocess.PIPE)
+    try:
+        # script that runs on uppmax
+        uq = subprocess.Popen(["/sw/uppmax/bin/projinfo", '-q'], stdout=subprocess.PIPE)
+    except Exception, e:
+        logging.error(e.message)
+        exit()
 
     # output is lines with the format: project_id  cpu_usage  cpu_limit
     output = uq.communicate()[0]
+
+    logging.info("CPU Hours Usage:")
+    logging.info(output)
```

Comment:
It's not much of an issue, but you could also write
logging.info("CPU Hours Usage : {0}".format(output))